server/src/main/java/com/server/loan/calculator/services/AdminService.java

```diff
@@ -0,0 +1,38 @@
+package com.server.loan.calculator.services;
+
+import com.server.loan.calculator.models.AdminData;
+import com.server.loan.calculator.validators.InputAsCorrectNumberValidator;
+import com.server.loan.calculator.validators.ValueInRangeValidator;
+import lombok.Data;
+import org.springframework.stereotype.Service;
+
+@Service
+@Data
+public class AdminService {
+
+    private final AdminData adminData;
+    private final ValueInRangeValidator valueInRangeValidator;
+    private final InputAsCorrectNumberValidator inputValidator;
+
+    private void validateAdminInput () {
```

Comment:
would make more sense to move this validation to a separate validator class like DataValidator